projects/orbit/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/google/orbit"
+language: c++
+primary_contact: "hebecker@google.com"
+auto_ccs :
+  - "orbitprofiler-eng+fuzztests@google.com"
```

Comment:
Note that people who are in this group and want to fix bugs will need to be added individually, since being in a googlegroup won't give them access to testcases.

Reply:
Ah, thanks for letting me know. There will for sure be others who need access. I will compile a list and make a follow-up PR.